src/osd/PrimaryLogPG.cc

```diff
@@ -5703,6 +5703,47 @@ int PrimaryLogPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
       }
       break;
 
+    case CEPH_OSD_OP_SET_REDIRECT:
```

Comment:
For now let's disallow this on anyother other than the base tier.  so, if pg_pool_t::is_tier() is true then EINVAL?  i would rather not worry about redirects interacting with cache tiering until we've throught it through carefully.